libraries/appbase/include/appbase/application.hpp

```diff
@@ -156,6 +156,18 @@ namespace appbase {
             return *ptr;
          }
 
+         template<typename Plugin>
+         bool is_plugin_initialized()const {
```

Comment:
Could something like `app().get_plugin<producer_plugin>().get_state()` work instead of adding this?

Reply:
Thanks Matt. Just tried and get_state worked.